test/truefi2/Liquidator2.test.ts

```diff
@@ -94,8 +94,9 @@ describe('Liquidator2', () => {
       standardTokenOracle: tusdOracle,
       creditModel,
     } = await setupTruefi2(owner, _provider))
-
-    loan = await createLoan(loanFactory, borrower, usdcPool, parseUSDC(1000), YEAR, 1000)
+    const legacyLoanTokenImpl = await new LegacyLoanToken2__factory(owner).deploy()
+    await loanFactory.setLoanTokenImplementation(legacyLoanTokenImpl.address)
+    loan = await createLoan(loanFactory, borrower, usdcPool, parseUSDC(1000), YEAR, 1000) as any
```

Comment:
Is there a way to avoid using `as any`?

Reply:
We can but this is a one-time case when createLoan returns LegacyLoanToken instead of LoanToken and it's due to changes done outside of the function. Since it's a one-off, I prefered to use as any to show exactly this.

Reply:
loan will still be typed as LegacyLoanToken